tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,11 @@
+Exit Code: 1
+Standard output:
+lib/ExternalModule.js(305,28): error TS2554: Expected 1 arguments, but got 0.
```

Comment:
`getUsedName` is used 3 times with no arguments, so it's likely intentional -- or perhaps a repeated error.

The usage is odd, though, so it's probably fine to force an optionality annotation.